helm/hpcc/templates/esp.yaml

```diff
@@ -144,6 +154,9 @@ kind: ConfigMap
 {{- end }}
 {{- end }}
 
-{{ include "hpcc.addCertificate" (dict "root" $ "name" .name "service" . "component" $application "visibility" .service.visibility) }}
+{{ include "hpcc.addCertificate" (dict "root" $ "name" .name "service" .service "component" $application "visibility" .service.visibility) }}
+{{- if $commonCtx.externalCert }}
```

Comment:
These 3 sections took me a while to work out.  May just want a comment explaining that these are to create and mount the local mtls certificate for the ESP when the service is public.